src/Entity/SecurityAdvisory.php

```diff
@@ -162,7 +168,14 @@ public function calculateDifferenceScore(RemoteSecurityAdvisory $advisory): int
         }
 
         if ($advisory->getTitle() !== $this->getTitle()) {
-            $score++;
+            $increase = 1;
+
+            // Do not increase the score if the title was just renamed to add a CVE e.g. from CVE-2022-xxx to CVE-2022-99999999
+            if (Preg::match('#^(CVE-[0-9]{4}-[x]+:)(.*)$#', $this->getTitle(), $matches) && str_contains($advisory->getTitle(), $matches[2])) {
```

Comment:
Adding support for more invalid CVE patterns, trying to be creative, hopefully this catches most placeholders people may use :)

```suggestion
            if (Preg::match('#^(CVE-[0-9x*?]{1,4}-?[0x*?]+:)(.*)$#', $this->getTitle(), $matches) && str_contains($advisory->getTitle(), $matches[2])) {
```